src/om_pouch/core.cljs

```diff
@@ -47,7 +47,7 @@
                       (update v :item/by-id merge {id params}))))))
   :pouchput ast})
 
-(defui Row
+(defui ^:once Row
```

Comment:
This prevents the def'd symbol from being redefined, but redefines all of the protocol implementations.  Reduces surprises during figwheel code reloads.